ci-operator/config/stolostron/foundation-e2e/stolostron-foundation-e2e-main.yaml

```diff
@@ -44,14 +44,6 @@ resources:
       memory: 200Mi
 test_binary_build_commands: make build GO_BUILD_FLAGS:='-race' --warn-undefined-variables
 tests:
-- as: pr-image-mirror
```

Comment:
why delete this?

Reply:
This repo does not need images like `2.8.0-PR2-f9c58a786ded99a9e2ec8b8b08cc6fe397184d81`